src/asmcnc/apps/drywall_cutter_app/widget_drywall_shape_display.py

```diff
@@ -727,9 +727,16 @@ def update_bumpers_and_validation_labels(self, current_shape, current_x, current
 
        # Now show a message if any dimensions are too big
         d_limit = self.X_MAX
-        if current_shape == 'circle' and float(self.d_input.text or 0) > d_limit:
-            self.d_input_validation_label.text = 'MAX: ' + str(d_limit)
-            self.d_input_validation_label.opacity = 1
+        if current_shape == 'circle':
+            if float(self.d_input.text or 0) > d_limit:
+                self.d_input_validation_label.text = 'MAX: ' + str(d_limit)
+                self.d_input_validation_label.opacity = 1
+            elif self.dwt_config.active_config.toolpath_offset.lower() == "inside" and \
+                float(self.d_input.text or 0) <= 0.1 + self.dwt_config.active_cutter.dimensions.tool_diameter:
```

Comment:
please change this to "< 0.2 ..." because this is the threshold we give for the min value:

e.g. 8mm cutter, set D to 8.11 and you  are fine because is it bigger than "cutter + 0.1" but still lower than the minimum.